docs/tutorial/fdfitting.rst

```diff
@@ -0,0 +1,343 @@
+example a new `Model` and `FdFit` is created at every cycle of the for loop.
+
+Statistically, it is typically more optimal to fit data using global fitting, as more
+information goes into estimates of parameters shared between different conditions. It's
+usually a good idea to think about which parameters you expect to be different between
+different experiments and only allow these parameters to be different. For example, if the
+only expected difference between different experiments is the contour length, then this
+can be achieved using::
+
+    odijk_inv = pylake.inverted_odijk("DNA")
+    odijk_fit = pylake.FdFit(odijk_inv)
+    for i, (distance, force) in enumerate(zip(distances, forces)):
+        odijk_fit.add_data(f"RecA {i}", force, distance, {"DNA/Lc": f"DNA/Lc_{i}"})
+    odijk_fit.fit()
+    print(odijk_fit.parameters)
+
+Note that this piece of code will lead to parameters `DNA/Lc_0`, `DNA/Lc_1` etc.
+
+Incremental fitting
+-------------------
+
+Fits can also be done incrementally::
+
+    >>> odijk_inv = pylake.inverted_odijk("DNA")
+    >>> odijk_fit = pylake.FdFit(odijk_inv)
+    >>> print(odijk_fit.parameters)
+    No parameters
+
+We can see that there are no parameters to be fitted. The reason for this is that
+we did not add any data to the fit yet. Let's add some and fit this data::
+
+    >>> data1 = odijk_fit.add_data("Control", f=f1, d=d1)
+    >>> odijk_fit.fit()
+    >>> print(odijk_fit.parameters)
+    Name         Value  Unit      Fitted      Lower bound    Upper bound
+    ------  ----------  --------  --------  -------------  -------------
+    DNA/Lp    59.409    [nm]      True                  0            inf
+    DNA/Lc     2.81072  [micron]  True                  0            inf
+    DNA/St  1322.9      [pN]      True                  0            inf
+    kT         4.11     [pN*nm]   False                 0              8
+
+Let's add a second dataset where we expect a different contour length and refit::
+
+    >>> data2 = odijk_fit.add_data("RecA", f=f2, d=d2, params={"DNA/Lc": "DNA/Lc_RecA"})
+    >>> print(odijk_fit.parameters)
+    Name              Value  Unit      Fitted      Lower bound    Upper bound
+    -----------  ----------  --------  --------  -------------  -------------
+    DNA/Lp         89.3347   [nm]      True                  0            inf
+    DNA/Lc          2.80061  [micron]  True                  0            inf
+    DNA/St       1597.68     [pN]      True                  0            inf
+    kT              4.11     [pN*nm]   False                 0              8
+    DNA/Lc_RecA     3.7758   [micron]  True                  0            inf
+    
+We see that indeed the second parameter now appears. We also note that the parameters
+from the first fit changed. If this was not intentional, we should have fixed
+these parameters after the first fit. For example, we can fix the parameter `DNA/Lp`
+by invoking::
+
+    >>> odijk_fit["DNA/Lp"].vary = False
```

Comment:
Not a big deal, but consider inverting this logic to `.fixed = True`. I just find it nicer to instruct someone to "Do `x` by setting `x = True`" rather than "Do `x` by setting `y = False`".

Reply:
I actually had a preference for `.fixed` as well. The reason I went with `vary` was to be compatible with `lmfit` in that regard as quite a few people might already be familiar with that library. Then again, we do quite different things in many places from `lmfit`, so we might as well switch to `.fixed`. I'll change it.